go.mod

```diff
@@ -74,6 +74,7 @@ require (
 )
 
 require (
+	github.com/gobuffalo/packr v1.30.1
```

Comment:
This should not be here.

Reply:
Ya, I had a rogue file: core/services/chainlink/a_chainlink-packr.go

Removed now